cloudinit/net/tests/test_dhcp.py

```diff
@@ -181,6 +232,13 @@ def test_parse_static_routes_returns_valid_routes_until_parse_err(self):
         logs = self.logs.getvalue()
         self.assertIn(rfc3442, logs.splitlines()[0])
 
+    def test_redhat_format(self):
+        redhat_format = "24.191.168.128 192.168.128.1,0 192.168.128.1"
```

Comment:
Are we certain that there's no whitespace after the comma?  If there can be, this will break

```
>>> dhcp.parse_static_routes("16.192.168.128 192.168.128.1, 0 192.168.128.1"
[('192.168.0.0/16', '128.192.168.128'), ('.0.0.0/1', '0.192.168.128')]

```

Reply:
hum good catch, I actually copy/pasted the lease in there and modifying the format.  In other words, I'm sure that there isn't a space in the one I have here.  

Still, making sure we skip over empty "token" would make sense.  it will be fix a moment. (I'll add the use case as a test as well).

Reply:
I consider this resolved as the code is now considering all tokens that are not empty (in other words, all tokens that have signification).